pkg/controller/installation/core_controller.go

```diff
@@ -186,7 +195,7 @@ type ReconcileInstallation struct {
 	status               status.StatusManager
 	typhaAutoscaler      *typhaAutoscaler
 	namespaceMigration   *migration.CoreNamespaceMigration
-	requiresTSEE         bool
+	tseeAPIAvailable     bool
```

Comment:
Hm, not sure about this - implies the apiserver is available to me. 

Maybe `runEnterpriseControllers`? or `enterpriseCRDsExist`?